keybinding.go

```diff
@@ -286,8 +286,12 @@ const (
 
 // Modifiers.
 const (
-	ModNone       Modifier = Modifier(0)
-	ModAlt                 = Modifier(tcell.ModAlt)
-	ModMouseShift          = Modifier(tcell.ModShift)
-	ModMouseCtrl           = Modifier(tcell.ModCtrl)
+	ModNone Modifier = Modifier(0)
+	ModAlt           = Modifier(tcell.ModAlt)
+
+	// ModShift only makes sense on keys that are not characters like the
+	// arrow keys and any mouse keys
+	// Character keys will instead be triggerd as their translated variant.
+	ModShift     = Modifier(tcell.ModShift)
+	ModMouseCtrl = Modifier(tcell.ModCtrl)
```

Comment:
I guess the `ModMouseShift` was added by you just recently, is that correct? Asking just to know if it makes sense to keep it around for backward compatibility. Or if it's here for short amount of time, then maybe it's not needed (as probably not many people would use it yet).

Reply:
Yes that is correct so it should not be released yet